### ISSUE TITLE
Bug Report: Volume of pico mixes aren't consistent

### ISSUE THREAD

**Issue by JackXson-Real** [FunkinCrew/Funkin#3247]
### Issue Checklist

- [X] I have properly named the issue
- [X] I have checked the issues/discussions pages to see if the issue has been previously reported

### What platform are you using?

Itch.io (Downloadable Build) - Windows

### If you are playing on a browser, which one are you using?

None

### Version

0.5.0

### Context (Provide images, videos, etc.)

Basically what the title says.

### Steps to reproduce (or crash logs, errors, etc.)

Play any pico mixes, for example: Fresh Pico is much quieter than Dadbattle, but its more than just those.

**Comment by AbnormalPoof** [comment]
Less of a bug and moreso how the songs are mixed.

**Comment by JackXson-Real** [comment]
> Less of a bug and moreso how the songs are mixed.

No some of the songs are very clearly much quieter than the other songs plus SFX. I actually have to increase my volume for some, but then I get earraped by the results screen SFX.

**Comment by JackXson-Real** [comment]
> Yeah, this thing is very unchecked by the devs but it's a real problem btw, cause I keep lowering and rising the volume every 2 minutes. Perfect Ranking music from Pico is way to loud for example and every time I end a song I have to lower it and then rise it again up to the next song. And also yeah, some songs vocals are sooo low, or sooo high and if you play with switched inst the vocals aren't even noticeable lol

Yeah they desperately need an automated system in-game that automatically adjusts the volume of every element to be the same as whatever your current in-game volume level is.
